### DISCUSSION TITLE
About "Rec. 709" vs "BT.709" naming convention.

### DISCUSSION THREAD

**Discussion by thomasmansencal** [colour-science/colour#704] [General]
This one has been itching me for a while and a discussion thread on 3D-Pro raised it up in the priority stack. Initially we have been using *Rec. 709*/*Rec. 2020* as colourspace names. Then when we split the transfer functions into their own sub-package, we started naming them using *BT.709*, *BT.2020*, **BT.1886** and this is the way I would have been keen on naming stuff for consistency.

Some examples names:

- *Rec. 709*
- *BT.709* <--- One I'm keen on using
- *ITUR_709* <--- Apple
- *BT709* <--- Google
- *ITU-RBT709* <--- ICC

**Comment by zachlewis** [comment]
Have you considered / are you aware of the nomenclature used in the official SMPTE Metadata Registers? They seem to be using "ITU709" and "ITU-R BT.709" for Symbols and Names, respectively: https://registry.smpte-ra.org/view/published/labels_view.html

(Probably for a different thread, but it couldn't hurt to have references to appropriate SMPTE URNs; i.e., urn:smpte:ul:060e2b34.04010101.04010101.01020000)

**Comment by thomasmansencal** [comment]
@zachlewis : Good find!

I have extracted a row for further discussion:

| Symbol                        | Namespace                            | UL                                               | Kind | Name                                 | Definition                                                                               | Applications | Notes | Defining Document | Is Deprecated |
|-------------------------------|--------------------------------------|--------------------------------------------------|------|--------------------------------------|------------------------------------------------------------------------------------------|--------------|-------|-------------------|---------------|
| TransferCharacteristic_ITU709 | http://www.smpte-ra.org/reg/400/2012 | urn:smpte:ul:060e2b34.04010101.04010101.01020000 | LEAF | ITU-R BT.709 Transfer Characteristic | Identifies ITU-R BT.709 transfer characteristic (also used in SMPTE 170M, 274M and 296M) |              |       | ITU-R BT.709      | false         |

My preference is for either:

- BT.709
- ITU-R BT.709

**Comment by meshula** [comment]
For the record, here's all of Apple's named color spaces.

https://developer.apple.com/documentation/coregraphics/cgcolorspace

The actual name is ITU-R BT.709

 https://www.itu.int/dms_pubrec/itu-r/rec/bt/R-REC-BT.709-6-201506-I!!PDF-E.pdf

I'm inclined to the long name.

**Comment by thomasmansencal** [comment]
Thanks Nick!
